src/main/kotlin/app/revanced/patches/youtube/misc/microg/shared/Constants.kt

```diff
@@ -4,4 +4,6 @@ object Constants {
     internal const val BASE_MICROG_PACKAGE_NAME = "com.mgoogle"
     internal const val REVANCED_PACKAGE_NAME = "app.revanced.android.youtube"
     internal const val REVANCED_APP_NAME = "YouTube ReVanced"
+    internal const val REVANCED_MUSIC_APP_NAME = "YT Music ReVanced"
```

Comment:
The constant is in the wrong package, it should be moved to the MicroG patch package

Reply:
yeah makes no sense mb

Reply:
done